installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -10,29 +10,49 @@ toc::[]
 
 In environments where the cloud identity and access management (IAM) APIs are not reachable, or the administrator prefers not to store an administrator-level credential secret in the cluster `kube-system` namespace, you can put the Cloud Credential Operator (CCO) into manual mode before you install the cluster.
 
+//Alternatives to storing administrator-level secrets in the kube-system project
 include::modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc[leveloffset=+1]
 
-[role="_additional-resources"]
-.Additional resources
+For a detailed description of all available CCO credential modes and their supported platforms, see xref:../../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc#about-cloud-credential-operator[About the Cloud Credential Operator].
+
+//Supertask: Configuring a Google Cloud Platform cluster to use short-term credentials
+[id="installing-aws-with-short-term-creds_{context}"]
+=== Configuring an Amazon Web Services cluster to use short-term credentials
+
+To install a cluster that is configured to use the Cloud Credential Operator (CCO) in manual mode with AWS Security Token Service (STS), you must configure the CCO utility and create the required AWS resources for your cluster.
+
+//Task part 1: Configuring the Cloud Credential Operator utility
+include::modules/cco-ccoctl-configuring.adoc[leveloffset=+3]
```

Comment:
I know you are reusing existing content, but this module lists the OpenShift CLI (oc) as a prerequisite and then one of the steps details how to obtain the release image version. If we think of an installation as a step-wise procedure, at no point prior have to told users to get these resources. I know modules make xrefs difficult, but I do think we should direct users to the doc that details how to do this. I do not think this is a big deal, but for a net new user, they might be left wondering how I get the prerequisite resources.

@bscott-rh please keep me honest, but I believe you would also need to download the installation media to complete the step here, right? 

https://60222--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/manually-creating-iam.html#cco-ccoctl-configuring_manually-creating-iam-aws

Reply:
Yes, the steps require the `openshift-install` binary so you'd have to download and extract that.

Reply:
Note to self: add the getting oc module to these assemblies until removing the assemblies entirely.